website/docs/tutorial/set-up-data-platform.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Set up your data platform
+id: set-up-data-platform
+description: "Set up your environment to get started using dbt."
+---
+
+Now that you have some context for this tutorial, let's get your environment set up. Check out the table of contents to the left to set up your environment on BigQuery, Snowflake, or Databricks. These guides are helpful for anyone with an existing data platform or starting from scratch.
```

Comment:
suggestion: let's break out the 3 options into bullets with direct links to the tutorial to make this super smooth.  Otherwise, learners will have to figure out that they need to use the ToC on the left hand side.